internal/api/db/db.go

```diff
@@ -0,0 +1,20 @@
+package db
+
+import (
+	"omshub/core-api/internal/api/db/models"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+func Init() *gorm.DB {
+	dsn := "host=localhost user=postgres password=postgres dbname=postgres port=5432 sslmode=disable"
```

Comment:
Pass the connection string as an argument

Reply:
done